src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
+$dayWidth: 100px;
+$borderWidth: 1px;
+$borderCount: 6;
+$marginSides: 10px;
+$calendar-days: ('mon' 1, 'tue' 2, 'wed' 3, 'thu' 4, 'fri' 5, 'sat' 6, 'sun' 7);
+$gap: 1px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh; /* 100% высоты видимого окна браузера */
+}
+
+.calendar {
+  width: calc((100 * 7) + (1 * 6) + (10 * 2));
+
+  /* (#{$dayWidth} * 7) + (#{$borderWidth} * #{$borderCount}) +
```

Comment:
better to use the variables here instead of some magic numbers

Reply:
the problem is, when i'm using variables, that lint writes me that i have a mistake